civihr_employee_portal/views/templates/views-view--my-details.tpl.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
not sure what's going on here. What view is this template for? Can you give more context please

Reply:
Sure, comment was added in the top of this file.